addon/package.json

```diff
@@ -65,9 +65,9 @@
     "@ember-data/serializer": ">= 3.28",
     "@ember-data/store": ">= 3.28",
     "@ember/string": ">= 3.0.0",
-    "@ember/test-helpers": "^2.9.3 || >= 3",
-    "active-model-adapter": "^3.0.1 || ^4.0.0",
-    "ember-data-model-fragments": "^5.0.0 || ^6.0.10",
+    "@ember/test-helpers": ">= 3",
+    "active-model-adapter": "^4.0.0",
+    "ember-data-model-fragments": "^6.0.10",
```

Comment:
In theory, no-one should have been on ember-data-model-fragments v5 since min ember-data is 3.28 and that is onlyt supported by EDMF v6+

So updating EDMF peer is mostly ceremony